browser/src/control/jsdialog/Widget.FormulabarEdit.js

```diff
@@ -106,7 +106,7 @@ function _appendCursor(cursorLayer) {
 }
 
 function _setSelection(cursorLayer, text, startX, endX, startY, endY) {
-	cursorLayer.innerHTML = '';
+	var newCursorLayer = document.createDocumentFragment();
```

Comment:
Surely we should use newCursorLayer in the code below to build the fragment? it seems we use cursorLayer (?)

Reply:
done